controllers/grid/users/reviewer/form/ResendRequestReviewerForm.php

```diff
@@ -45,21 +44,54 @@ public function __construct(ReviewAssignment $reviewAssignment, ReviewRound $rev
             $submission,
             'controllers/grid/users/reviewer/form/resendRequestReviewerForm.tpl'
         );
+
+        // Validation checks for this form
+        $this->addCheck(new \PKP\form\validation\FormValidator($this, 'responseDueDate', 'required', 'editor.review.errorAddingReviewer'));
+        $this->addCheck(new \PKP\form\validation\FormValidator($this, 'reviewDueDate', 'required', 'editor.review.errorAddingReviewer'));
```

Comment:
Should we also add `FormValidatorPost` to the parent class? (not sure about that)